tests/rkt_net_test.go

```diff
@@ -200,6 +200,68 @@ func TestNetDefaultNetNS(t *testing.T) {
 	f("")
 }
 
+func TestNetGWv4Connectivity(t *testing.T) {
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	f := func(argument string) {
+		httpPort, err := testutils.GetNextFreePort4()
+		if err != nil {
+			t.Fatalf("%v", err)
+		}
+		httpServeAddr := fmt.Sprintf("0.0.0.0:%v", httpPort)
+		httpServeTimeout := 30
+		iface := "eth0"
```

Comment:
This variable `iface` is both used for designating an interface on the host (in the call to `testutils.GetGWv4`) and an interface in the pod (in the `/inspect` command).

I don't think it is correct to use "eth0" on the host: in my case, I have wifi "wlp3s0" on my laptop but not always "eth0".